test/commands.js

```diff
@@ -564,6 +564,58 @@ let runFundCrowdsaleBelowSoftCap = async (command, state) => {
   return state;
 }
 
+let runFundCrowdsaleOverSoftCap = async (command, state) => {
+  if (!state.crowdsaleFinalized) {
+    // unpause the crowdsale if needed
+    if (state.crowdsalePaused) {
+      state = await runPauseCrowdsaleCommand({pause: false, fromAccount: state.owner}, state);
+    }
+
+    // set weiPerUSDinTGE rate if needed
+    if (state.weiPerUSDinTGE == 0) {
+      state = await runSetWeiPerUSDinTGECommand({wei: 10000, fromAccount: state.owner}, state);
+    }
+
+    let softCap = await state.crowdsaleContract.maxFoundationCapUSD.call(),
```

Comment:
same here, cant it be calculated outside the contract?

Reply:
Same response :)